ocf_data_sampler/load/gsp.py

```diff
@@ -28,4 +30,4 @@ def open_gsp(zarr_path: str | Path) -> xr.DataArray:
 
     )
 
-    return ds.generation_mw
+    return get_xr_data_array_from_xr_dataset(ds, datavar='generation_mw')
```

Comment:
I think we shouldn't use this function here. For GSP data I think we can assume that we will always call it `generation_mw` and we are hard coding the name here anyway by setting the `datavar` parameter. I'd reverse this change